### ISSUE TITLE
inf_test failed

### ISSUE THREAD

**Issue by ni1lloc** [baidu-research/warp-ctc#152]
I'm trying to compile warp-ctc on CentOS 7, both cpu_test and gpu_test fails.
The cost in inf_test is a positive float. Have tried adding softmax before loss function, but it didn't help.
latest master, compiled with GCC 4.8.5
![image](https://user-images.githubusercontent.com/17943172/71880039-69d3bd00-316a-11ea-9052-94560558de63.png)

**Comment by light1003** [comment]
I got the same error,did you sovle it?

**Comment by zoidburg** [comment]
Met the same problem with the master branch. I checked the v1 stable version tests, the cpu tests passed but the gpu tests failed in the small_test case. 

Running GPU tests
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error: compute_ctc_loss in small_test, stat = execution failed
Aborted (core dumped)
